Source/ui_unix/mainwindow.cpp

```diff
@@ -309,8 +309,7 @@ void MainWindow::saveState()
     ui->menuLoad_States->actions().at(m_stateSlot-1)->setText(QString("Load Slot %1 - %2").arg(m_stateSlot).arg(datetime));
 }
 
-void MainWindow::loadState(){
-    int m_stateSlot = sender()->property("stateSlot").toInt();
+void MainWindow::loadState(int m_stateSlot){
```

Comment:
Prefixing names with `m_` is reserved for class member variables. This should be `stateSlot`.